packages/jest-snapshot/src/index.ts

```diff
@@ -384,7 +385,7 @@ const _toMatchSnapshot = (config: MatchSnapshotConfig) => {
           `must be explicitly passed to write a new snapshot.\n\n` +
           `This is likely because this test is run in a continuous integration ` +
           `(CI) environment in which snapshots are not written by default.\n\n` +
-          `Received:${actual.includes('\n') ? '\n' : ' '}${RECEIVED_COLOR(
+          `Received:${actual.includes('\n') ? '\n' : ' '}${getReceivedColor()(
```

Comment:
I still don't understand why this needs to be an extra function call? Can this be `withReceivedColor(actual)` or something, and that can on the fly check `chalk.level`? the `thing()(otherThing)` looks weird to me

Reply:
I guess it's due to the `getSnapshotColor()` further along. I still think it'd be preferable to keep it at a minimum, but I guess keeping the differences between the apis down is good as well

Reply:
Oh yeah, good question. I was thinking about “lazy evaluation” in case the tests pass and we never need the colors for a report.

I am happy to simplify to more intuitive: `bReceivedColor(actual)`

```js
export const aSnapshotColor = getSnapshotColorForChalkInstance(chalk);
export const bReceivedColor = getReceivedColorForChalkInstance(chalk);
```

Reply:
I'd prefer that :)